configs/ubuntu16_04_clang/BUILD

```diff
@@ -54,3 +54,33 @@ docker_toolchain_autoconfig(
     tags = ["manual"],
     test = True,
 )
+
+# Created on 2018.06.08
+# Container: gcr.io/cloud-marketplace/google/clang-ubuntu@sha256:d553634f23f7c437ca35bbc4b6f1f38bb81be32b9ef2df4329dcd36762277bf7
+docker_toolchain_autoconfig(
+    name = "ubuntu16_04-clang-0.1.0-bazel_0.14.1-autoconfig",
+    additional_repos = ubuntu16_04_clang_default_repos(),
+    base = "@ubuntu16_04-clang//image",
+    bazel_version = "0.14.1",
+    env = clang_env(),
+    keys = ubuntu16_04_clang_default_keys(),
+    packages = ubuntu16_04_clang_default_packages(),
+    tags = ["manual"],
+    test = True,
+)
+
+# Created on 2018.06.08
+# Container: gcr.io/cloud-marketplace/google/clang-ubuntu@sha256:d553634f23f7c437ca35bbc4b6f1f38bb81be32b9ef2df4329dcd36762277bf7
+docker_toolchain_autoconfig(
```

Comment:
0.1.0 --> 1.0

Reply:
All done. Since this is only change the bazel target names that generate the configs, there is no need to update configs etc.